single-line-diagram-core/src/test/resources/NodeDecoratorsBranchStatusNodeBreaker.svg

```diff
@@ -812,6 +812,9 @@ polyline {fill: none}
             <circle cx="5" cy="15" r="5"/>
             <circle cx="11" cy="15" r="5"/>
             <circle cx="8" cy="19" r="5"/>
+            <g class="sld-flash">
+                <path d="M 5.5,14.5 8.5,6.36 4.32,5.83 4.2,0.3 1.3,8.47 5.4,8.99 Z" transform="translate(3.0,-0.5)"/>
+            </g>
```

Comment:
This flash should not be added... It's due to using Middle3WTNode for two different usages: it is either a node among `voltageLevelGraph.getNodes()` or a multiTermNode of a `SubstationGraph` or `VoltageLevelGraph`.  We should display a flash only in the first case, and to know you're in this case you need to test if `middle3wtNode.getGraph() != null` (in `DefaultDiagramLabelProvider::getNodeDecorators`).